lib/raygun.ts

```diff
@@ -355,11 +397,30 @@ class Raygun {
       body: req.body,
     };
 
-    this.send(err, customData || {}, requestParams, [
-      "UnhandledException",
-    ]).catch((err) => {
-      console.error("[Raygun] Failed to send Express error", err);
-    });
+    // If a local store of breadcrumbs exist in the response
+    // run in scoped breadcrumbs store
+    if (res.locals && res.locals.breadcrumbs) {
+      breadcrumbs.runWithBreadcrumbs(() => {
```

Comment:
Obtain the scoped breadcrumbs from the response locals as explained in the previous comment. They are provided to the send method using the `runWithBreadcrumbs` scoping method.